compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "LLVMWrapper.h"
 
+#include "llvm/Analysis/AliasAnalysis.h"
```

Comment:
The NewPM support has turned into a big mess because there were major API changes in all of the recent releases. After the LLVM 12 upgrade, we should probably look into switching to NewPM by default (as clang will be doing so soon as well), and drop support for NewPM on older LLVM versions to reduce the mess.